README.md

```diff
@@ -4,7 +4,7 @@
 [![JetBrains incubator project](https://jb.gg/badges/incubator.svg)](https://confluence.jetbrains.com/display/ALL/JetBrains+on+GitHub)
 [![GitHub license](https://img.shields.io/badge/license-Apache%20License%202.0-blue.svg?style=flat)](https://www.apache.org/licenses/LICENSE-2.0)
 
-**Kover** - Gradle plugin for Kotlin code coverage agents: [IntelliJ](https://github.com/JetBrains/intellij-coverage)
+**Kover** - Gradle plugin for Kotlin code coverage tools: [Kover](https://github.com/JetBrains/intellij-coverage)
```

Comment:
While I'm OK with Engine -> Tool renaming, I'm not sure people would understand the difference between Kover (Gradle plugin) and Kover (Coverage tool). It may confuse people when we try to explain to them they can use Kover with Jacoco Tool and Kover with Kover tool. Also, the repository name is still `intellij-coverage`. Have you considered just naming it `Intellij Tool`?

Reply:
The repository is also used for IDEA, and at the moment the benefits of renaming it are doubtful, and it will be quite problematic.

The mention of `IntelliJ` in the name of the tool is also a bit misleading, because it has no direct relation to the `IntelliJ platform`.

I see the main development prospect that the Carpet should become an independent project, independent of the way it is used. That is, whether it is a Gradle Plugin, Kotlin Compiler plugin or CLI or maven in the future, they all just have to use the Kover.

In this case, the main problem is that the plugin and the tool will be called the same. In JaCoCo, it is called similarly, but its plugin supports only one tooling.

Might be worth doing some extra design on that?